### PR TITLE
Add support for `--iframes`, `--pdf-script`, and `--pdf-event-script`

### PR DESCRIPTION
**Pull request by adrianwong**

### CODE REVIEW

File: src/main/java/com/princexml/wrapper/AbstractPrince.java [hunk 1]
```diff
@@ -679,6 +692,31 @@ public void setPdfOutputIntent(String pdfOutputIntent) {
         this.pdfOutputIntent = pdfOutputIntent;
     }
 
+    /**
+     * Include an AcroJS script to run when the PDF is opened.
+     * @param pdfScript The filename or URL of the AcroJS script.
```
Comment (mikeday) [inline src/main/java/com/princexml/wrapper/AbstractPrince.java:697]: And `null` to clear?

**Review by mikeday — APPROVED** [review]
Looks fine!

File: src/main/java/com/princexml/wrapper/AbstractPrince.java (hunk 1 above)
Comment (adrianwong) [reply src/main/java/com/princexml/wrapper/AbstractPrince.java:697]: We never explicitly mention this in other setters that take a string. I'm fine with leaving it out; I assume someone using this library should figure it out quickly enough.

Comment (mikeday) [reply src/main/java/com/princexml/wrapper/AbstractPrince.java:697]: Sounds fair!

**Pull request merged**